airbyte-integrations/connectors/source-mysql/src/main/java/io/airbyte/integrations/source/mysql/initialsync/MySqlInitialLoadRecordIterator.java

```diff
@@ -99,8 +99,8 @@ protected AirbyteRecordData computeNext() {
     if (isCdcSync && cdcInitialLoadTimeout.isPresent()
         && Duration.between(startInstant, Instant.now()).compareTo(cdcInitialLoadTimeout.get()) > 0) {
       final String cdcInitialLoadTimeoutMessage = String.format(
-          "Initial load for table %s has taken longer than %s, Canceling sync so that CDC replication can catch-up on subsequent attempt, and then initial snapshotting will resume",
```

Comment:
Unfortunately the `getTableName()` in the parent is `Optional.empty(). As a result, the sync ends up failing with a system error (NPE) instead of a transient error. See https://cloud.airbyte.com/workspaces/2946ed1c-bc9d-47df-881d-82d29c91d4a8/connections/fc7d7915-f484-443b-999b-c81f23138d01/job-history 

I think it's fine as it's easily identifiable from the logs + state which table failed. Plus, I think it's something the user doesn't particularly care about

Reply:
Oh oops, this is mysql. Will remove this